src/main/java/com/cloudbees/dockerpublish/DockerBuilder.java

```diff
@@ -266,63 +260,9 @@ private boolean recordException(Exception e) {
     	
     }
 
-    @Override
-    public DescriptorImpl getDescriptor() {
-        return (DescriptorImpl)super.getDescriptor();
-    }
-
-    /**
-     * Descriptor for {@link DockerBuilder}. Used as a singleton.
-     * The class is marked as public so that it can be accessed from views.
-     *
-     * <p>
-     * See <tt>src/main/resources/hudson/plugins/hello_world/DockerBuilder/global.jelly</tt>
-     * for the actual HTML fragment for the plugin global config screen.
-     */
     @Extension
     public static final class DescriptorImpl extends BuildStepDescriptor<Builder> {
 
-        public String getUserName() {
-            return userName;
-        }
-        public String getPassword() {
-            return password;
-        }
-        public String getEmail() { return email; }
-        public String getRegistryUrl() { return registryUrl; }
-
-
-        private String userName;
-        private String password;
-        private String email;
-
-
-        private String registryUrl;
```

Comment:
the credentials are not migrated as noted in Readme (and major version increase). The registry endpoint is